github/github.go

```diff
@@ -134,8 +134,8 @@ func (c *Config) NewClient() (*Client, error) {
 			}
 		}
 	}
-
-	client.UserAgent = "actions-runner-controller"
+	version := getVersion("ARC_VERSION", "dev")
+	client.UserAgent = "actions-runner-controller:" + version + ""
```

Comment:
Can we instead use the `product/version` pattern as I believe it's relatively more [standard-ish](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent)?

```suggestion
	client.UserAgent = "actions-runner-controller/" + version
```

Reply:
Yeah that is fine i just choose the image:tag was because it was the first to come to mind